cpu/samd21/include/periph_cpu.h

```diff
@@ -35,6 +35,26 @@ enum {
 };
 
 /**
+ * @brief   Override GPIO modes
+ *
+ * We use 3 bit to determine the pin functions:
+ * - bit 0: PU or PU
+ * - bit 1: input enable
+ * - bit 2: pull enable
+ * @{
+ */
+#define HAVE_GPIO_MODE_T
+typedef enum {
+    GPIO_IN    = 0x2,           /**< IN */
```

Comment:
change these to use preprocessor macros or some other kind of symbolic name to avoid magic numbers in the implementation.